### PR TITLE
Add post, LPN overview Part 1: INFODEV-1875

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-f8f0502925/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-f8f0502925/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-d4e8e54277/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-be3cae3678/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-49ed57efa5/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-49ed57efa5/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-4dfb6a2f33/)

**Comment by RackerKate** [comment]
@cyrichardson For your review please.

**Comment by cyrichardson** [comment]
On it, @RackerKate.

**Comment by cyrichardson** [comment]
@RackerKate Suggested changes here: 
[blog_An overview of LPN and shortcuts to mobile Telnet- Part I_cyrEdits.docx](https://github.com/rackerlabs/docs-developer-blog/files/2069940/blog_An.overview.of.LPN.and.shortcuts.to.mobile.Telnet-.Part.I_cyrEdits.docx)

**Comment by RackerKate** [comment]
@catlook For your review please.

**Comment by catlook** [comment]
on it!

File: _posts/2018-06-04-lpn-overview-part1.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+date: 2018-06-04 00:00
+comments: true
+author: Sharad Kukreja
+published: true
+authorIsRacker: true
+categories:
+    - General
+---
+
+In this two-part series we'll discuss License Plate Numbers (LPN) in
```
Comment (catlook) [inline _posts/2018-06-04-lpn-overview-part1.md]: ... series, we'll ..

File: _posts/2018-06-04-lpn-overview-part1.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+Oracle&reg; Warehouse Management (WMS), why and how to use them, as well as
```
Comment (catlook) [inline _posts/2018-06-04-lpn-overview-part1.md]: ... Warehouse Management System (WMS) ...

File: _posts/2018-06-04-lpn-overview-part1.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+  item controls.
+
+### LPN context and status codes
+
+LPNs have the following context and status codes:
+
+**Resides in Inventory**: The material associated with the LPN has been costed
+and accounted for in an inventory. An LPN with this context may not be used
+when receiving material against a standard or inspection-routed receipt, but
+may be used during a direct delivery-routed receipt. Outbound transactions can
```
Comment (catlook) [inline _posts/2018-06-04-lpn-overview-part1.md]: it may be used ...

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-ad9348b922/). :bow:

**Pull request merged**